C/plugins/storage/postgres/connection.cpp

```diff
@@ -749,7 +772,7 @@ bool Connection::fetchReadings(unsigned long id, unsigned int blksize, std::stri
 char	sqlbuffer[200];
 
 	snprintf(sqlbuffer, sizeof(sqlbuffer),
-		"SELECT id, asset_code, read_key, reading, user_ts AT TIME ZONE 'UTC' as \"user_ts\", ts AT TIME ZONE 'UTC' as \"ts\" FROM foglamp.readings WHERE id >= %ld LIMIT %d;", id, blksize);
+		"SELECT id, asset_code, read_key, reading, user_ts AT TIME ZONE 'UTC' as \"user_ts\", ts AT TIME ZONE 'UTC' as '|ts\" FROM foglamp.readings WHERE id >= %ld ORDER BY id LIMIT %d;", id, blksize);
```

Comment:
I got the error  : 
`Jan 23 16:07:13 foglamp FogLAMP Storage[26252]: PostgreSQL storage plugin raising error: ERROR:  unterminated quoted string at or near "'|ts" FROM foglamp.readings WHERE id >= 1 ORDER BY id LIMIT 5;"#012LINE 1: ...ZONE 'UTC' as "user_ts", ts AT TIME ZONE 'UTC' as '|ts" FROM...#012                                                             ^
`
it should be the '|ts\"